Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -254,8 +258,15 @@ static std::optional<DefaultInterface> GetSystemDefaultInterface()
                               get_addr(iface->ifa_broadaddr)};
     }
   }
+#else
```

Comment:
It would be tidier to invert the `!defined(__ANDROID__)` condition above and put the Android code below the Windows code.